src/17-infinite-discovery/helpers.ts

```diff
@@ -11,8 +11,8 @@ import { fromPairs, defaults } from "lodash"
  *
  * https://drive.google.com/drive/u/1/folders/1Lh7msUc0R_JlpNEzApZ4YbqB8x5tbpws
  */
-export async function getArtworks() {
-  const filePath = path.join(__dirname, "./data/artworks.json")
+export async function getArtworks(filename: string) {
+  const filePath = path.join(__dirname, `./data/${filename}`)
```

Comment:
**suggestion:** Maybe a filename could be optional here? Also this method is being used [here](https://github.com/artsy/quantum//blob/b9ced7bd39fc97cc3c687e04e40c68c4f3978e39/src/17-infinite-discovery/02-artworks.ts#L20).